src/stories/temp-converter.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from "react";
+
+export type TempConverterProps = {
+    items: string[];
+};
+
+export default function TempConverter(props: TempConverterProps) {
+    const [startUnit, setStartUnit] = useState<string>("");
+    const [endUnit, setEndUnit] = useState<string>("");
+    const [temperature, setTemperature] = useState<string>("");
+    const [convertedTemp, setConvertedTemp] = useState<string>("");
+
+    const isDisabled = !temperature || !startUnit || !endUnit;
+
+    const convertTemperature = (value: number, from: string, to: string) => {
+        if (from === to) return value;
+
+        if (from === "Celsius" && to === "Fahrenheit")
```

Comment:
Please do not use inline if statements, but use blocks for better readability.

Also I already said that in a previous comment, but since you are using the literal values of the units here, it is not only a suboptimal choice to make the units a property, but the component would literally not work anymore if you would pass ANY other values than these 3. Just to point the reason out a bit more.